### DISCUSSION TITLE
Fully Render Distant Textures

### DISCUSSION THREAD

**Discussion by ScoFoZo** [open-goal/jak-project#2884] [Jak 1]
Is there a way to do this? I'd like to see the whole game in full resolution from the top of the Forbidden Forest or Snowy Mountain or the final boss... And maybe a way to hide the UI of the spyglass for screenshots. I know this would be graphics intensive, but it would be very cool for screenshots.

**Comment by Zedb0T** [comment]
The game has a built in free camera mode you can use to get screenshots, but the games engine is limited to loading a set certain amount of levels per game.     Also the all the levels include low poly versions of levels in the distance so you can see them on console - when you "see" low poly misty it isn't actually misty being rendered low poly but a part of the village level.

**Reply by ScoFoZo** [reply]
So someone would have to swap all those polys out for the real deal in each zone? Gotchya
